app/src/utils/apiHelpers.ts

```diff
@@ -190,3 +193,22 @@ export const getLaneData = async ({
   const apiUrl = `${process.env.API_URL}/api/v2/collections?genre=${slug}&page=${pageNum}&per_page=${perPage}`;
   return await fetchApi(apiUrl);
 };
+
+export const getSearchData = async ({
```

Comment:
Can you add esdoc style comments and include that this is used on the `/search/index` page? Just to associate a page route with the relevant Repo API endpoint.